File: src/main/java/com/aws/greengrass/localdebugconsole/messageutils/DeviceDetails.java
/*
 * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 * SPDX-License-Identifier: Apache-2.0
 */

package com.aws.greengrass.localdebugconsole.messageutils;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@AllArgsConstructor
@NoArgsConstructor
public class DeviceDetails {
    private String os;
    private String version;
    private String cpu;
    private String rootPath;
    private String logStore;
    private Boolean registered;
    private String thingName;
}

File: src/main/js/dashboard-frontend/src/util/DeviceDetails.ts
/*
 * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 * SPDX-License-Identifier: Apache-2.0
 */

export interface DeviceDetails {
  os: string;
  version: string;
  cpu: string;
  rootPath: string;
  logStore: string;
  registered: boolean;
  thingName: string;
}
